service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/OrderIssuedEmailContentProvider.java

```diff
@@ -11,67 +11,36 @@
-import static uk.gov.service.notify.NotificationClient.prepareUpload;
 
 @Slf4j
 @Service
 public class OrderIssuedEmailContentProvider extends AbstractEmailContentProvider {
     private final HmctsCourtLookupConfiguration config;
-    private final RepresentativeService service;
 
     @Autowired
     protected OrderIssuedEmailContentProvider(@Value("${ccd.ui.base.url}") String uiBaseUrl,
                                               ObjectMapper mapper,
-                                              HmctsCourtLookupConfiguration config,
-                                              RepresentativeService service) {
+                                              HmctsCourtLookupConfiguration config) {
         super(uiBaseUrl, mapper);
-        this.service = service;
         this.config = config;
     }
 
-    public Map<String, Object> buildNotificationParametersForHmctsAdmin(final CaseDetails caseDetails,
-                                                                        final String localAuthorityCode,
-                                                                        final byte[] documentContents,
-                                                                        final IssuedOrderType issuedOrderType) {
-        CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
-        List<Representative> representativesServedByPost = service.getRepresentativesByServedPreference(
-            caseData.getRepresentatives(), POST);
-        List<String> formattedRepresentatives = formatRepresentativesForPostNotification(representativesServedByPost);
-
-        return ImmutableMap.<String, Object>builder()
-            .put("callout", (issuedOrderType != NOTICE_OF_PLACEMENT_ORDER) ? buildCallout(caseData) : "")
-            .put("needsPosting", isNotEmpty(representativesServedByPost) ? "Yes" : "No")
-            .put("doesNotNeedPosting", representativesServedByPost.isEmpty() ? "Yes" : "No")
-            .put("courtName", config.getCourt(localAuthorityCode).getName())
-            .putAll(caseUrlOrDocumentLink(isNotEmpty(representativesServedByPost), documentContents,
-                caseDetails.getId()))
-            .put("respondentLastName", getFirstRespondentLastName(caseData.getRespondents1()))
-            .put("representatives", formattedRepresentatives.isEmpty() ? "" : formattedRepresentatives)
-            .build();
-    }
-
-    public Map<String, Object> buildNotificationParametersForRepresentatives(final CaseDetails caseDetails,
-                                                                             final String localAuthorityCode,
-                                                                             final byte[] documentContents,
-                                                                             final IssuedOrderType issuedOrderType) {
+    //For email served representatives (users without case access)
```

Comment:
personally i believe that it is redundant, client should decide how it is used

Reply:
removed comments